grpc/server/src/engine_server/mappings/ipc/execute_request.rs

```diff
@@ -41,11 +41,17 @@ impl TryFrom<ipc::ExecuteRequest> for ExecuteRequest {
 
         let protocol_version = request.take_protocol_version().into();
 
+        // TODO: it is currently unclear what the expectation is to provide a proposer when using
+        // the execution engine in standalone mode.
+        let proposer =
```

Comment:
Why not let user decide who's the proposer? Burning the funds by default might be surprising behavior

Reply:
I'd say in standalone mode the gas cost should always be 0, i.e. no payment should be made?

Reply:
per Henry, we will revisit this later as part of a larger question about continuing support of standalone EE.